source/reference/program/mongostat.txt

```diff
@@ -179,15 +179,21 @@ average operations per second.
    megabytes. This is the total data size at the time of the last
    :program:`mongostat` call.
 
-.. describe:: size
+.. describe:: vsize
 
    The amount of virtual memory in megabytes used by the process at
    the time of the last :program:`mongostat` call.
 
 .. describe:: non-mapped
 
-   The total amount of virtual memory excluding all mapped memory at
+   .. versionchanged:: 3.0.0 
+   Only for :ref:`storage-mmapv1`.
```

Comment:
does this compile correctly without the blank line in between versionchanged and the next line?